src/attributes/attr.js

```diff
@@ -113,8 +126,13 @@ boolHook = {
 		if ( value === false ) {
 			// Remove boolean attributes when set to false
 			jQuery.removeAttr( elem, name );
+		} else if ( getSetInput || !ruseDefault.test( name ) ) {
+			elem.setAttribute( jQuery.propFix[ name ] || name, name );
+
 		} else {
-			elem.setAttribute( name, name );
+			// Support: IE<9
```

Comment:
See https://contribute.jquery.org/style-guide/js/#comments, we will fix them all eventually, but i guess new code should adhere to it already